client/packages/common/src/intl/locales/en/common.json

```diff
@@ -392,7 +394,7 @@
   "messages.no-scanners-found": "No scanners found",
   "messages.not-applicable": "N/A",
   "messages.not-initialised": "[ Not configured ]",
-  "messages.over-allocated": "Due to the pack sizes available a total quantity of {{quantity}} has been allocated rather than {{issueQuantity}}",
+  "messages.over-allocated": "Due to the pack sizes available a total quantity of {{allocatedQuantity}} has been allocated rather than {{requestedQuantity}}",
```

Comment:
offroad - a bug that I'd fixed in the plugin branch and hadn't realised it wasn't in develop